markdown/report.Rmd

```diff
@@ -62,7 +136,7 @@ glossary2 <- c("Palliative care is an approach that improves the quality of life
 glossary <- data.frame(glossary1, glossary2)
 ```
 
-```{r MetaData, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
+```{r metadata, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE}
```

Comment:
Can we automate the last published, next published and date form completed dates? e.g.

For the last published date:

```
format(pub_date, "%d/%m/%Y")
```

We can define the next published and date form completed dates in 00_setup-environment.R and automate here?